boards/arm/google_kukui/google_kukui_defconfig

```diff
@@ -15,11 +15,16 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
+CONFIG_SHELL=y
```

Comment:
Enabling shell in the board defconfig is not recommended, this is something that can be enabled in the application, you can also `tests/boards/board_shell` which enabled different shell modules.
1 test is failing because of the thread being created by the shell in a no-multi-threading test.

Reply:
OK I see. I dropped that patch